### PR TITLE
Bug 1998378: Use reference to extension models instead of kind

### PR DESCRIPTION
**Pull request by yaacov**
Screenshots:
Before:
screenshot-localhost_9000-2021 11 01-16_07_16

After:
screenshot-localhost_9000-2021 11 01-15_59_06

### CODE REVIEW

**Comment by yaacov** [comment]
@glekner @avivtur please review

File: frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/health.ts [hunk 1]
```diff
@@ -33,29 +33,36 @@ type KubevirtHealthResponse = {
 export const additionalResources = {
   vmis: {
     isList: true,
-    kind: VirtualMachineInstanceModel.kind,
-  },
-  pods: {
-    isList: true,
-    kind: PodModel.kind,
+    kind: referenceForExtensionModel({
```
Comment (glekner) [inline frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/health.ts:36]: you can replace all of these with `kubevirtReferenceForModel`

e.g. `kubevirtReferenceForModel(VirtualMachineInstanceModel)`

**Review by glekner — CHANGES_REQUESTED** [review]

**Pull request merged**